### DISCUSSION TITLE
03. Formulaire de contact

### DISCUSSION THREAD

**Discussion by dNicolle** [cnumr/best-practices-wordpress#15] [Bonnes pratiques] [answered]
Pour la fiche : https://github.com/cnumr/fiches-wordpress-greenit/blob/main/best-practices-WordPress/fiches/03.%20Formulaires%20de%20contact.md

L'intitulé de cette fiche ne me semble pas être au bon format, ce n'est pas une recommandation, mais un sujet.

Je propose de séparer cette fiche en 2 inclure des recommandations dans les titres : 
 * Limiter le nombre de données récoltées par formulaire en ligne (rgpd)
 * Sécuriser les formulaires en ligne (données stockées, captcha)
 
J'en mettrais bien une 3ième sur le choix entre ne pas mettre de formulaire du tout et en mettre en suivant les recommandations des 2 précédentes.

**Comment by hrenaud** [comment]
> Le troisième choix va à l'encontre de la première phrase de la fiche :D

Je suis assez d'accord pour découper en deux fiches, une troisième ne me semble pas opportune, car ce pourrait être juste une reco qui pourrait être ajouté dans les deux autres.

**Comment by dNicolle** [comment, marked as answer]
J'ai créé les deux nouvelles fiches, mais je n'ai pas encore pris le temps de les rédiger.
Je supprimerais la fiche 03 quand j'aurais fait les deux autres.
